packages/library-base/src/commonMain/kotlin/io/realm/kotlin/internal/Converters.kt

```diff
@@ -50,8 +53,8 @@ import kotlin.reflect.KClass
  * main abstraction of conversion used throughout the library.
  */
 internal interface RealmValueConverter<T> {
-    public fun publicToRealmValue(value: T?): RealmValue
-    public fun realmValueToPublic(realmValue: RealmValue): T?
+    fun MemTrackingAllocator.publicToRealmValue(value: T?): RealmValue
+    fun MemTrackingAllocator.realmValueToPublic(realmValue: RealmValue?): T?
```

Comment:
This doesn't need to track buffers, does it?

Reply:
And it is equally fast with the `master` getter-benchmarks? 👀

Reply:
I missed this one.